src/drivers/heater/heater.cpp

```diff
@@ -129,6 +125,17 @@ void Heater::Run()
 
 void Heater::initialize_topics()
 {
+	// Auto-detect primary accelerometer ID
```

Comment:
That assumes that the primary is heated, which is not a valid assumption. As @davids5 suggested I think it's better if you allow the user to provide the bus the heated unit sits on (or the actual sensor). And if that unit fails (goes offline) the heating should shut down as well. I'm not seeing that logic in here yet and it could lead to overheating the IMU unit.